packages/react/src/primitives/SelectField/SelectField.tsx

```diff
@@ -10,6 +10,27 @@ import { SelectFieldProps, Primitive } from '../types';
 import { splitPrimitiveProps } from '../shared/styleUtils';
 import { useStableId } from '../shared/utils';
 
+const selectFieldChildren = (
+  children?: React.ReactNode,
+  options?: string[]
```

Comment:
These arguments should be extracted into separate interface, and the options type value should reference `SelectFieldProps['options']` so as to keep the parent component and this function signature in sync.